nodeup/pkg/model/docker.go

```diff
@@ -653,8 +653,8 @@ var dockerVersions = []dockerVersion{
 		Hash:          "0a1325e570c5e54111a79623c9fd0c0c714d3a11",
 		ExtraPackages: map[string]packageInfo{
 			"container-selinux": {
-				Version: "2.68",
-				Source:  "http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.68-1.el7.noarch.rpm",
+				Version: "2.107",
+				Source:  "http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.107-3.el7.noarch.rpm",
```

Comment:
would be good to also let people override "http://mirror.centos.org/" so we can override our own mirror locations.